backend/games/tests/test_leaderboard.py

```diff
@@ -0,0 +1,79 @@
+from datetime import datetime
+from games.models.scores import Score
+from backend.tests_utils import WeakAuthenticationBaseTestCase
+from django.db.models import Sum
+
+
+class LeaderboardTestCase(WeakAuthenticationBaseTestCase):
+    fixtures = ["test_authentication.yaml", "games.yaml", "test_scores.yaml"]
+
+    EXPECTED_FIELDS = {"user", "total_score"}
+
+    def endpoint_list(self):
+        return "/games/scores/leaderboard/"
+
+    def list(self):
```

Comment:
`list` is a very generic term, it would be easier to read with something like `query_list_endpoint`